Testscripts/Windows/PERF-SYSCALL-BENCHMARK.ps1

```diff
@@ -117,58 +112,56 @@ collect_VM_properties
                             $finalResult += $currentResult
                             $metadata = "test=$testType"
                             $syscallResult = "AverageReal=$avgReal AverageUser=$avgUser AverageSystem=$avgSystem"
-                            $resultSummary +=  New-ResultSummary -testResult "$syscallResult : Completed" -metaData $metadata -checkValues "PASS,FAIL,ABORTED" -testName $currentTestData.testName
```

Comment:
is it not needed?

Reply:
no need, if added, it will show a PASS under test summary